Makefile

```diff
@@ -1,28 +1,57 @@
-all: init build up
+NAME = dina-web/collections-data-model
+VERSION = $(TRAVIS_BUILD_ID)
+ME = $(USER)
+HOST = beta.dina-web.net
+TS := $(shell date '+%Y_%m_%d_%H_%M')
+
 
+all: init build up
 
 init:
-	@echo "Pulling code from GitHub"
-	# cd cco-tools && git clone https://github.com/Inkimar/cco_poc.git
-	cd cco-tools && git clone https://github.com/DINA-Web/cco_poc.git
+	mkdir -p mysql-datadir backups
 
 build:
+	@echo "Pulling source code for dependencies from GitHub"
+	cd cco-tools && git clone https://github.com/DINA-Web/cco_poc.git
+	cd cco-tools && curl -L -s -o wait-for-it.sh \
+		https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh && \
+		chmod +x wait-for-it.sh
 	docker-compose build cco
 
 up: 
-	@echo "Sorry for the 30s below, please fix it with a PR ;)"
+	@echo "Starting database and creating schema..."
 	docker-compose up -d db
-	sleep 30 && docker-compose run cco
+	docker-compose run cco
 
 connect:
-	@echo "Connecting to CCO database and display the tables"
-	mysql -u dina -ppassword12 -P 13306 -h 127.0.0.1 cco -e "show tables"
+	@export $(cat .env | xargs) > /dev/null
+	@echo "Connecting to db... pls enter 'show tables' at the prompt."
+	docker-compose run --rm db sh -c \
```

Comment:
Sure, it actually already does that, I think. It starts up a fresh container to do just what you describe with the twist that it then removes it. 

Using `docker-compose run` instead of `docker exec` means I don't need to have a fully qualified container name (I can use "db" instead as defined in the compose-file instead of the fully qualitifed dwcco_db_1) and the `--rm` removes it immediately after. 

So there is already no dependency on `mysql` on the host machine... The one in the image is being used. You can see a slight difference in the layout/look'n'feel if you switch from mariadb to mysql - both provide mysql clients but the mariadb variant looks slightly